app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :user, foreign_key: 'user_id'
+  belongs_to :post, foreign_key: 'post_id'
+
+  # The function `update_post_likes_counter` increments the `LikesCounter`
+  # attribute of a post.
```

Comment:
- [ ] It would be nice to add the `after_save :update_likes_counter` method to update the database after a like has been added 👍🏿